index.html

```diff
@@ -1,160 +1,278 @@
             </li>
-
-
+            <li>
+              <a href="#" class="about_icon"
+                ><img src="./image/twitter_icon.png" alt="twitter_icon"
+              /></a>
+            </li>
+            <li>
+              <a href="#" class="about_icon"
+                ><img src="./image/m_icon.png" alt="m_icon"
+              /></a>
+            </li>
+          </ul>
+    
+          <a href="#" class="button2">Get my resume</a>
+        </div>       
+        <ul class="about_group">
+          <li>
+            <div class="lang_tech">
+              <h3 class="lang_title">Languages</h3>
+              <img class="lang_arrow" src="./image/Disabled.png" alt="" />
+            </div>
+  
+            <ul class="about_flex">
+              <li class="about_skill">
+                <img class="about_img" src="./image/JS_icon.png" alt="" />
+                <h4 class="about_lang">JavaScript</h4>
+              </li>
+  
+              <li class="about_skill">
+                <img class="about_img" src="./image/html_icon.png" alt="" />
+                <h4 class="about_lang">HTML</h4>
+              </li>
+  
+              <li class="about_skill">
+                <img class="about_img" src="./image/css_icon.png" alt="" />
+                <h4 class="about_lang">CSS</h4>
+              </li>
+            </ul>
+          </li>
+  
+          <li class="lang_tech border">
+            <h3 class="lang_title">Frameworks</h3>
+            <img class="lang_arrow" src="./image/narrow.png" alt="" />
+          </li>
+  
+          <li class="lang_tech border">
+            <h3 class="lang_title">Skills</h3>
+            <img class="lang_arrow" src="./image/narrow.png" alt="" />
+          </li>
         </ul>
+      </section>
+  
+      <section class="sectionForm">
+        <div class="form">
+            <h2 class="form-title">Contact me</h2>
+            <p class="form-parragraph">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I'd love to help with it </p>
+          <form class="form-form" action="https://formspree.io/f/xlevbkay" method="post">
+            <ul class="ul-form">
+              <input class="formInput"
+                type="text"
+                name="name"
+                placeholder="Full name"
+                maxlength="30"
+                required
+                />
+              <input class="formInput"
+                  type="email"
+                  name="email"
+                  placeholder="Email address"
+                  required
+              />
+              <textarea class="formArea"
+                  type="text"
+                  name="message"
+                  rows="5"
+                  maxlength=""
+                  placeholder="Write your message here"
+                  required
+              ></textarea>
+              <button class="button-form" type="submit">Get in touch</button>
+            </ul>
```

Comment:
- [ ] it would be better if you give the same height and width to the inputs in order to match the Figma design 
- [ ] Also you should add some padding to the inputs so it will be the same as in the Figma design 
-Yours 
<img width="470" alt="Screen Shot 2022-10-01 at 22 14 52" src="https://user-images.githubusercontent.com/61790587/193428827-323c9229-e74d-4d85-977a-a71ddccc861b.png">
-Figma 
<img width="421" alt="Screen Shot 2022-10-01 at 22 22 03" src="https://user-images.githubusercontent.com/61790587/193428844-7e3ecd41-e6d4-4c1b-9f59-e2312a089540.png">